docs/UserGuide.adoc

```diff
@@ -18,160 +18,598 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+29/06/2019, the deadline at 23:59 of 02/07/2019. The reminder will
+remind you at these two timings.
+
+=== Editing reminder: `reminder edit`
+
+Format: *reminder edit INDEX [t/TIME_DATE] [n/NOTE]*
+
+=== Deleting reminder: `reminder delete`
+
+Deletes the the specified reminder.
+
+Format: *reminder delete INDEX*
+
+=== Sorting bookmarks: `sort`
+
+Sorts the displayed bookmarks according to different criteria (name,
+order added) and displays them as a list.
+
+Format: *sort [CRITERIA]*
+
+* CRITERIA is the field you use to sort. Acceptable values of CRITERIA
+are: *name* and *default*.
+ * CRITERIA is case insensitive, e.g. both URL and url are acceptable
+criteria.
+
+Examples:
+
+* *sort name* +
+Lists all the bookmarks based on name/alphabetical order.
+
+* *sort default* +
+Sort all the bookmarks based on the order they were added.
+
+=== Expanding folder hierarchy: `expand`
+
+Expands the folder hierarchy by the specified number of levels.
+
+Format: *expand [LEVEL]*
+
+* LEVEL should be an integer. If it is more than zero, folder hierarchy
+will expand by LEVEL number of levels. If LEVEL is less than zero,
+folder hierarchy will collapse by the given number of levels. If LEVEL
+is zero, nothing happens.
+ * Current level of display + LEVEL must be positive and less than or equal
+to the maximum number of levels
+ * If LEVEL is negative, the folder hierarchy will be collapsed by the
+specified number of levels
+ * If LEVEL is not specified, expand to the maximum number of levels.
+
+Examples
+
+* *expand 3*
+
+=== Collapsing folder hierarchy: `collapse`
+
+Collapses the folder hierarchy by the specified number of levels.
+
+Format: *collapse [LEVEL]*
+
+* LEVEL should be an integer. If it is more than zero, folder hierarchy
+will collapse by LEVEL number of levels. If LEVEL is less than zero,
+folder hierarchy will expand by the given number of levels. If LEVEL is
+zero, nothing happens.
+ * Current level of display + LEVEL must be positive and less than or equal
+to the maximum number of levels of the folder hierarchy
+ * If LEVEL is negative, the folder hierarchy will be collapsed by the
+specified number of levels
+ * If LEVEL is not specified, collapse to show only one level of folders.
+
+Examples
+
+* *collapse 1*
+
+=== Switching link:#faq[[.underline]#tabs#]: `tab`
+
+Switches the current app view to the tab that corresponds to the given
+index.
+
+Format: *tab INDEX*
+
+* INDEX is 1, 2 or 3, which correspond to the
+link:#faq[[.underline]#Dashboard#], Online, and Offline tabs
+respectively.
+
+Examples
+
+* *tab 1* +
+Goes to the Dashboard tab
+
+* *tab 2* +
+Goes to the Online tab which is a web view
+
+* *tab 3* +
+Goes to the Offline tab which shows the offline copy of bookmarks
 
 == FAQ
 
-*Q*: How do I transfer my data to another Computer? +
-*A*: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Address Book folder.
+*What are Tabs?*
```

Comment:
We should probably make the FAQs headings as well, so we can link to them directly.